src/main/resources/templates/order.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Order</title>
+</head>
+<body>
+
+<table class="orderTable" style="display: none" >
+    <tr>
+        <th>ID</th>
+        <th>Prepared</th>
+        <th>Item name</th>
+        <th>Item price</th>
+        <th>Item number</th>
+    </tr>
+</table><br>
+<button type="button" id="showData">Show orders</button>
+<button type="button" id="editData">edit order</button>
+<button type="button" id="deleteData">delete order</button>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+    <script>
+     var showOrders = 0;
+     $("#showData").click(function(){
+     showOrders++;
+         if(showOrders == 1){
```

Comment:
Als je dit met een click wil doen, is het misschien handiger om de hele table te clearen en daarna opnieuw te pullen. Het hele idee van zo'n knop is vaak dat je geupdate informatie erin krijgt.